test/module/irohad/torii/torii_service_test.cpp

```diff
@@ -612,11 +608,12 @@ TEST_F(ToriiServiceTest, FailedListOfTxs) {
                      != iroha::protocol::TxStatus::STATELESS_VALIDATION_FAILED
                  and --resub_counter);
 
-        auto error_beginning = toriiResponse.error_message().substr(
-            0, toriiResponse.error_message().find_first_of('.'));
-
         ASSERT_EQ(toriiResponse.tx_status(),
                   iroha::protocol::TxStatus::STATELESS_VALIDATION_FAILED);
-        ASSERT_EQ(error_beginning, error_msg_beginning);
+        auto msg = toriiResponse.error_message();
+        ASSERT_EQ(
```

Comment:
Please use the following instead, it looks more obvious:
https://github.com/hyperledger/iroha/blob/ec9c10acf4273cd49d857a77fa84cc4001ea8b68/test/module/shared_model/validators/container_validator_test.cpp#L104-L105